src/directives/for.js

```diff
@@ -95,8 +95,8 @@ function evaluateItemsAndReturnEmptyIfXIfIsPresentAndFalseOnElement(component, e
     let items = component.evaluateReturnExpression(el, iteratorNames.items, extraVars)
 
     // This adds support for the `i in n` syntax.
-    if (isNumeric(items) && items > 0) {
-        items = Array.from(Array(items).keys(), i => i + 1)
+    if (isNumeric(items)) {
+        items = items > 0 ? Array.from(Array(items).keys(), i => i = 1) : []
```

Comment:
Why not: `items = Array.from(Array(items).keys(), i => i + 1)`?

Shouldn't need the ternary at all.

Also doesn't changing `+` to `=` cause a breaking change where each item in the iteration isn't incremented, but just set to the number `1`?

Please fix, then I'll merge

Reply:
Also, surprised the tests passed with the `=`, maybe we need an extra test to ensure that ever item is an incrementing value

Reply:
@calebporzio It should be `i + 1`, my laptop's keyboard is broken so I must have hit `=` instead of `+` in the suggestion.

The `i => i + 1` logic is just to start the counter from 1 instead of 0 so I'm guessing tests are still passing because we're not checking that the first item is `1`.

Reply:
That was my initial thought, just make it `items >= 0`. But I found it cleaner with the if/else or ternary. Personal preference. Will make the change as suggested. Negative value will error then, which is fine I guess.